examples/mach_cad_examples/example_induction_motor_jmag.py

```diff
@@ -0,0 +1,182 @@
+import os
+import sys
+
+# change current working directory to file location
+os.chdir(os.path.dirname(__file__))
+# add the directory immediately above this file's directory to path for module import
+sys.path.append("../..")
+
+import mach_cad.tools.jmag as JMAG
+from mach_cad.tools.femm import FEMM
```

Comment:
I suggest that we remove line 10 (importing FEMM)

Reply:
@elsevers sounds good, just made the changes.